common/rpc/interceptor/telemetry.go

```diff
@@ -62,19 +62,24 @@ var (
 
 // static variables used to emit action metrics.
 var (
-	respondWorkflowTaskCompleted = "RespondWorkflowTaskCompleted"
+	respondWorkflowTaskCompleted = "RespondWorkflowTaskCompletedRespondWorkflowTaskCompleted"
```

Comment:
is `RespondWorkflowTaskCompleted` meant to be repeated?

Reply:
No. Must be caused by some unintentional key strokes before I sent PR. :(